scripts/prepare-en-US.mjs

```diff
@@ -0,0 +1,54 @@
+import fs from "node:fs";
+import path from "node:path";
+
+const source_language_locale = "en-GB"
+const target_language_locale = "en-US"
```

Comment:
I personally think it would be better do do this as an object/associative array, so that we could it change in the future to have stuff like this:
```suggestion
c
const languageMapping = {
    'de-DE': ['de-AT', 'de-CH'],
    'en-GB': ['en-US'],
};
```

I wouldnt implement other dialects yet, but I guess we could get ready for that :D